src/_nebari/stages/kubernetes_kuberhealthy/__init__.py

```diff
@@ -0,0 +1,130 @@
+import contextlib
+import os
+import pathlib
+import sys
+from typing import Any, Dict, List, Type
+
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+
+from _nebari.apply_from_yaml import create_from_yaml
+from _nebari.stages.base import NebariTerraformStage
+from nebari import schema
+from nebari.hookspecs import NebariStage, hookimpl
+
+
+class InputSchema(schema.Base):
+    pass
+
+
+class OutputSchema(schema.Base):
+    pass
+
+
+class KubernetesTestStage(NebariTerraformStage):
```

Comment:
Maybe this could be an excellent example to play with `kustomize` as well, we could use a `KustomizeStage` class instead of the terraform one (I think you had one in your other PR) and test both.